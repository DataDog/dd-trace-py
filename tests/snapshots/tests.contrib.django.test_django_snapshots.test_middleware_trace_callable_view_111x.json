[[
  {
    "name": "django.request",
    "service": "django",
    "resource": "GET tests.contrib.django.views.FeedView",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "web",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "django",
      "django.request.class": "django.core.handlers.wsgi.WSGIRequest",
      "django.response.class": "django.http.response.HttpResponse",
      "django.user.is_authenticated": "False",
      "django.view": "feed-view",
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://testserver/feed-view/",
      "language": "python",
      "runtime-id": "771ab9d0147b4fa282ea8b1a6b7e0e3a",
      "span.kind": "server"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 2447
    },
    "duration": 11001000,
    "start": 1633584289662289000
  },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.contrib.sessions.middleware.SessionMiddleware.process_request",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 178000,
       "start": 1633584289662771000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.middleware.common.CommonMiddleware.process_request",
       "trace_id": 0,
       "span_id": 3,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 110000,
       "start": 1633584289663310000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.middleware.csrf.CsrfViewMiddleware.process_request",
       "trace_id": 0,
       "span_id": 4,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 95000,
       "start": 1633584289663484000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.contrib.auth.middleware.AuthenticationMiddleware.process_request",
       "trace_id": 0,
       "span_id": 5,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 49000,
       "start": 1633584289663639000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.contrib.auth.middleware.SessionAuthenticationMiddleware.process_request",
       "trace_id": 0,
       "span_id": 6,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 38000,
       "start": 1633584289663748000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.contrib.messages.middleware.MessageMiddleware.process_request",
       "trace_id": 0,
       "span_id": 7,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 99000,
       "start": 1633584289663844000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.middleware.security.SecurityMiddleware.process_request",
       "trace_id": 0,
       "span_id": 8,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 35000,
       "start": 1633584289664008000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.middleware.csrf.CsrfViewMiddleware.process_view",
       "trace_id": 0,
       "span_id": 9,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 44000,
       "start": 1633584289664214000
     },
     {
       "name": "django.view",
       "service": "django",
       "resource": "tests.contrib.django.views.FeedView",
       "trace_id": 0,
       "span_id": 10,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 7643000,
       "start": 1633584289664349000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.middleware.security.SecurityMiddleware.process_response",
       "trace_id": 0,
       "span_id": 11,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 48000,
       "start": 1633584289672102000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.middleware.clickjacking.XFrameOptionsMiddleware.process_response",
       "trace_id": 0,
       "span_id": 12,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 55000,
       "start": 1633584289672213000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.contrib.messages.middleware.MessageMiddleware.process_response",
       "trace_id": 0,
       "span_id": 13,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 42000,
       "start": 1633584289672327000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.middleware.csrf.CsrfViewMiddleware.process_response",
       "trace_id": 0,
       "span_id": 14,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 39000,
       "start": 1633584289672429000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.middleware.common.CommonMiddleware.process_response",
       "trace_id": 0,
       "span_id": 15,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 55000,
       "start": 1633584289672526000
     },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.contrib.sessions.middleware.SessionMiddleware.process_response",
       "trace_id": 0,
       "span_id": 16,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "component": "django"
       },
       "duration": 49000,
       "start": 1633584289672639000
     }]]
