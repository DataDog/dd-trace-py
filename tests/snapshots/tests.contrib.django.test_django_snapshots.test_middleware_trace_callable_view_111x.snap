[[{"name" "django.request"
   "service" "django"
   "resource" "GET tests.contrib.django.views.FeedView"
   "type" "web"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1612560766293833000
   "duration" 3966000
   "meta" {"runtime-id" "93145384bb364e9c90d9a9309fc26349"
           "django.view" "feed-view"
           "http.method" "GET"
           "django.response.class" "django.http.response.HttpResponse"
           "http.url" "http://testserver/feed-view/"
           "django.request.class" "django.core.handlers.wsgi.WSGIRequest"
           "http.status_code" "200"
           "django.user.is_authenticated" "False"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "_dd.tracer_kr" 1.0
              "system.pid" 96813}}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.contrib.sessions.middleware.SessionMiddleware.process_request"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294087000
       "duration" 109000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.middleware.common.CommonMiddleware.process_request"
       "error" 0
       "span_id" 2
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294245000
       "duration" 86000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.middleware.csrf.CsrfViewMiddleware.process_request"
       "error" 0
       "span_id" 3
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294362000
       "duration" 23000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.contrib.auth.middleware.AuthenticationMiddleware.process_request"
       "error" 0
       "span_id" 4
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294410000
       "duration" 25000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.contrib.auth.middleware.SessionAuthenticationMiddleware.process_request"
       "error" 0
       "span_id" 5
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294459000
       "duration" 10000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.contrib.messages.middleware.MessageMiddleware.process_request"
       "error" 0
       "span_id" 6
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294491000
       "duration" 53000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.middleware.security.SecurityMiddleware.process_request"
       "error" 0
       "span_id" 7
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294567000
       "duration" 15000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.middleware.csrf.CsrfViewMiddleware.process_view"
       "error" 0
       "span_id" 8
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294681000
       "duration" 24000}
      {"name" "django.view"
       "service" "django"
       "resource" "tests.contrib.django.views.FeedView"
       "error" 0
       "span_id" 9
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766294751000
       "duration" 2480000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.middleware.security.SecurityMiddleware.process_response"
       "error" 0
       "span_id" 10
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766297273000
       "duration" 18000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.middleware.clickjacking.XFrameOptionsMiddleware.process_response"
       "error" 0
       "span_id" 11
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766297317000
       "duration" 30000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.contrib.messages.middleware.MessageMiddleware.process_response"
       "error" 0
       "span_id" 12
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766297372000
       "duration" 17000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.middleware.csrf.CsrfViewMiddleware.process_response"
       "error" 0
       "span_id" 13
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766297411000
       "duration" 14000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.middleware.common.CommonMiddleware.process_response"
       "error" 0
       "span_id" 14
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766297446000
       "duration" 27000}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.contrib.sessions.middleware.SessionMiddleware.process_response"
       "error" 0
       "span_id" 15
       "trace_id" 0
       "parent_id" 0
       "start" 1612560766297497000
       "duration" 23000}]]
